recipes/imagepy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "imagepy" %}
+{% set version = "0.16" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "511556789365fffa4827f771ca246d10bbb7fd7a07240d6a29c39b3e44e83e1e" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash_value }}
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
```

Comment:
Apologies. The example is a bit outdated. We should fix that. Currently we encourage [using `pip` for the install]( python -m pip install --no-deps --ignore-installed ).